Modules/Quotas/src/Quotas.Infrastructure.Database.SqlServer/Migrations/20230628132645_MetricStatus.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Quotas.Infrastructure.Database.SqlServer.Migrations
+{
+    /// <inheritdoc />
+    public partial class MetricStatus : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropColumn(
+                name: "IsExhaustedUntil",
+                table: "TierQuotas");
+
+            migrationBuilder.AlterColumn<string>(
+                name: "MetricKey",
+                table: "TierQuotaDefinitions",
+                type: "nvarchar(50)",
+                maxLength: 50,
+                nullable: true,
```

Comment:
Why is this column nullable? It shouldn't be, right?

Reply:
I don't even understand why this was added. I removed this from the Up and Down, recreated the database and created another migration... which came up clean.

Reply:
Of course it came up clean, because you probably didn't change the Snapshot.
You should properly delete the migration (via the CLI), then do the fixes necessary for it to not create this code during the next migration generation.